doc/Options/z390_options.md

```diff
@@ -0,0 +1,131 @@
+SYSDAT(dir) | M | pgm dir | Define directory for retrieving DAT source input files for AREAD.  Set to SYSDAT(.) for current directory when no path is specified on DSNAME parm for AREAD.
+SYSERR(dir) | MALE | pgm dir | Define directory for output ERR files. This option may also override file name and/or suffix (*.sfx overrides just the suffix in the pgm directory).
+SYSLKD(file) | L | pgm dir | Define input source file for linker commands such as INCLUDE, ALIAS, ENTRY, and NAME.
+SYSLOG(dir) | E | pgm dir | Define directory for output LOG file. This option may also override file name and/or suffix (*.sfx overrides just the suffix in the pgm directory).
+SYSLST(dir) | AL | pgm dir | Define directory for output LST files.  This option may also override file name and/or suffix (*.sfx overrides just the suffix in the pgm directory).
+SYSMAC(dir) | M | mac dir | Define one or more directories for source MAC files.  If the option starts with + the directories listed will be appended to the maclib list.  Multiple directories can be specified in one parameter and are always separated by +. When you specify this option more than once and want previous parameter instructions to remain in place, ensure that you start parameter with a `+` otherwise it will drop prior directories.  This option may also override suffix by adding *.sfx.
+SYSMLC(dir) | MA | pgm dir | Define directory for source input MLC files.
+SYSOBJ(dir) | AL | pgm dir | Define directory for OBJ relocatable object files.
+SYSOPT(dir) | MALE | pgm dir | Define directory for @file option files.  This option may override suffix using *.sfx.
+SYSPARM('text') | M | none | Define text string which can be accessed by mz390 global macro variable &SYSPARM.
+SYSPCH(dir) | M | pgm dir | Define directory for PCH output files from PUNCH.  This option may also override file name and/or suffix (*.sfx overrides just the suffix in the pgm directory).  Note PUNCH output is in ASCII source format and is not directed to the OBJ file.  These files may be used as linker input commands via SYSLKD(file) option.  Set to SYSPCH(.) for current directory when no path is specified on DSNAME parm for PUNCH.
+SYSPRN(dir) | A | pgm dir | Define directory for assembler listing PRN output files. This option may also override file name and/or suffix (*.sfx overrides just the suffix in the pgm directory).
+SYSTERM(file) | MALE | pgm.ERR | Define alternate file for all start/end messages plus any error messages, plus related source file statistics for each file containing errors.  The default is the program name with ERR suffix.
+SYSTRC(dir) | MALE | pgm dir | Define directory for output TRACE TR? files.
+TEST | E | NO | Start execution with interactive TEST mode active which prompts for commands such as T opcode or G opcode to trace or execution to the next occurrence of opcode name.
+TEST(ddname) | E | NO | Define input command file to run TEST commands. _ddname_ is an environment variable set to the name of the command file.
```

Comment:
I've forgotten what my comment was about "_ddname_" last time. I don't think the underscores should be there at all. It may have been a way to indicate underlining -- I don't know. Ask Abe -- he should know what's correct.

Reply:
Please remove the underscores. They're confusing. The test() option accepts a DDNAME indeed, which in the z390 setup refers to a host environment variable pointing to the actual file.

Reply:
Underscores are a markdown format to indicate _italics_. They are not shown in the formatted output. Happy to remove.

Reply:
OK.. great.. resolving this item and leaving the _italics_.